Lisp/syntax_test_lisp.lisp

```diff
@@ -87,6 +87,9 @@
 (nil)
 ;^^^ constant.language
 
+()
+;^^^ constant.language
```

Comment:
I expect this test to fail, because the ^ are not aligned correctly. They should be directly underneath the parens:
```
 ()
;^^ constant.language
```

Reply:
Apologies, it's fixed.